lib/trace-of-tab.js

```diff
@@ -144,7 +138,6 @@ class TraceOfTab {
     const timestamps = {
       navigationStart: navigationStart.ts,
       firstPaint: getTimestamp(firstPaint),
-      firstContentfulPaint: firstContentfulPaint.ts,
```

Comment:
not sure it makes sense to remove fcp from here if we're leaving in `firstMeaningfulPaint`, `firstPaint`, etc

can we just remove the `if (!firstContentfulPaint` line?

Reply:
All of those are not needed - they are dropped on the floor in Tracium. I didn't touch them since there's an implicit goal to stay close to lighthouse's original code.

> can we just remove the if (!firstContentfulPaint line?

There are a few places down there that assume that `firstContentfulPaint` is not null - I removed those as well.

Reply:
should've just been the difference between using `getTimestamp` and `maybeGetTiming` but alright :)

Reply:
Ah, who knew there's `maybeGetTiming`!..